platform/ui/src/components/SidePanel/SidePanel.jsx

```diff
@@ -0,0 +1,131 @@
+import React, { useState, useCallback } from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+
+import { Button, Icon } from '@ohif/ui';
+
+const borderSize = 4;
+const expandedWidth = 320;
+const collapsedWidth = 60;
+
+const baseStyle = {
+  maxWidth: `${expandedWidth}px`,
+  width: `${expandedWidth}px`,
+};
+
+const styleMap = {
+  open: {
+    left: { marginLeft: '0px' },
+    right: { marginRight: '0px' },
+  },
+  closed: {
+    left: { marginLeft: `-${expandedWidth - collapsedWidth - borderSize}px` },
+    right: { marginRight: `-${expandedWidth - collapsedWidth - borderSize}px` },
+  },
+};
+
+const baseClassName =
```

Comment:
Some components we adopt the naming for base classes for `const baseClasses = ''`, what do you think? We can later go through all components to standardize them as well.

Reply:
Updated.